paasta_tools/mesos_tools.py

```diff
@@ -78,6 +78,22 @@ def raise_cli_exception(msg):
 from mesos.cli import master  # noqa
 import mesos.cli.cluster  # noqa
 
+# Works around a mesos-cli bug ('MesosSlave' object has no attribute 'id' - PAASTA-4119).
+# The method below gets a slave by its ID. Original code here:
+# https://github.com/mesosphere/mesos-cli/blob/master/mesos/cli/master.py#L176
+# uses "in" instead of "==", matching when the wanted ID is a substring of
+# the candidate. Because of that, multiple slaves are returned and mesos-cli
+# finds itself in a buggy "if" branch (hence the AttributeError).
+import itertools  # noqa
```

Comment:
I'm surprised flake8 is allowing us to have these random imports in the middle of the code? I guess the monkeypatch makes a difference when the import happens?